docs/iris_archive.md

```diff
@@ -1,31 +1,38 @@
-# Iris Archive
+# IRIS Archive
 
 ## Overview
 All files sent out across IRIS will additionally be stored within the IRIS Archive. The archive requires no authentication allowing quick data retrieval.
 
-Our CDN implements caching of files accessed within the past year and file compression to again speed up the service.
+The archive implements caching of files accessed within the past year and file compression to again speed up the service.
+
+## IRIS Archive Structure 
+
+All files are stored within a folder called `data`.
+
+Inside this we see a folder for each dataset, for example `ABUC`.
 
 ## Azure Storage Explorer
 
-To easily browse the IRIS Archive with a graphic interface, Azure Storage Explorer can be used.
+To easily browse the IRIS Archive with a graphical, Azure Storage Explorer can be used.
```

Comment:
I think this should be `graphical interface` rather than just `graphical` :)

Reply:
You are very right